templates/repo/pulls/tab_menu.tmpl

```diff
@@ -14,4 +14,10 @@
 		{{$.locale.Tr "repo.pulls.tab_files"}}
 		<span class="ui small label">{{if .NumFiles}}{{.NumFiles}}{{else}}-{{end}}</span>
 	</a>
+	<span class="item gt-ml-auto gt-pr-0 gt-font-bold gt-df gt-ac gt-gap-3">
+		<span><span class="text green">+{{if .Diff.TotalAddition}}{{.Diff.TotalAddition}}{{else}}0{{end}}</span> <span class="text red">-{{if .Diff.TotalDeletion}}{{.Diff.TotalDeletion}}{{else}}0{{end}}</span></span>
+		<span class="diff-stats-bar">
+			<div class="diff-stats-add-bar" style="width: 50%;"></div>
```

Comment:
This percentage is still pending?

Reply:
Confirmed

Reply:
Need to convert to css class？Need your help

Reply:
Use `Eval` like done here:

https://github.com/go-gitea/gitea/blob/a95a7bcdf30505150c8c94e1700599fcfad9f98e/templates/repo/diff/stats.tmpl#L4

Reply:
Is this expected?
![image](https://github.com/go-gitea/gitea/assets/89133723/7019da8c-6d9b-4c27-ba7a-4cb384708252)

Reply:
yes 80% would be expected here.